cmake_modules/docMacros.cmake

```diff
@@ -99,33 +99,47 @@ MACRO(DOCBOOK_TO_PDF _xsl _file _name)
         ENDIF(MAKE_DOCS)
 ENDMACRO(DOCBOOK_TO_PDF targetname_suffix srcfile outfile targetdir deps_list)
 
-MACRO(DOCBOOK_TO_HTML _xsl_file _xml_file _out_dir)
+MACRO(DOCBOOK_TO_HTML _xsl_file _xml_file _out_dir _html_target _css_path _zip_target)
     IF(MAKE_DOCS)
-       set(_html_zip "doc_generate_html_zip")
-       set(_html_files "_generate_html_files")
+       if("${_target_file}" STREQUAL "")
+           set(_tareget "_generate_html_files")
```

Comment:
Looks like _target is spelled wrong (_tareget) Not certain if it was intentional or if it matters.